### PR TITLE
Add support for DAX language

### PR DESCRIPTION
**Pull request by StanislawSwierc**
I've been successfully using highlight.js in my blog. Recently I started blogging about DAX and I noticed that there are no syntax highlighting rules for this language so I've added it. Please notice that DAX is really constrained to the available functions, thus, I classified them as keywords.

### CODE REVIEW

File: src/languages/dax.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    case_insensitive: true,
+    contains: [
+      hljs.C_LINE_COMMENT_MODE,
+      hljs.C_BLOCK_COMMENT_MODE,
+      {
+        begin: IDENT_RE,
+        lexemes: IDENT_RE,
+        keywords: {
+          keyword:
+            'ABS ACOS ACOSH ADDCOLUMNS ADDMISSINGITEMS ALL ALLEXCEPT ALLNOBLANKROW ' +
```
Comment (revosw) [inline src/languages/dax.js]: Just wanted to mention that you'll need to have some relevance filled out, so one solution is to add relevancy to your keywords, for instance:
`ABS|0 ACOS|0 ADDCOLUMNS|10 ADDMISSINGITEMS|10 ALL|10 ALLEXCEPT|10 ALLNOBLANKROW|10`

When the Travis CI build fails on your language, you need to be more relevant. When the Travis CI build misdetect another language for your language, you need to be less relevant. It's a good thing you set relevancy to 0 on strings, numbers and identifiers, now you just need to set relevancy to 10 on the keywords, and you're good to go mate. You can see the details of the build process and for what language it failed to detect at by either clicking "Details" or the red X by the change in the pr changelog

**Comment by StanislawSwierc** [comment]
I guess I pushed wrong branch. I'm fixing it now.

**Comment by StanislawSwierc** [comment]
ping

**Comment by yyyc514** [comment]
@StanislawSwierc 

**First of all, truly sorry for the long delay in anyone responding to this PR.**

Thank you so much for desiring to contribute and writing a language grammar.  Unfortunately, we currently are no longer accepting new languages into the core repository.  The existing maintainers simply don't have time to maintain new languages (fixing bugs, dealing with issues, etc).  **We'd still love to host your language as a 3rd party module though** - if you're willing to help maintain it.  

It's easy to turn your language into a module that anyone who likes can easily integrate with Highlight.js using a single line of code:

```html
<script src="/path/to/highlight.min.js"></script>
<!-- they'd simply add your language modules -->
<script src="/path/to/highlightjs-dax.js"></script> 
<script>hljs.initHighlightingOnLoad();</script>
```

It's also possible for users to rebuild the `highlight.js` library to include specific 3rd party modules, all in a single file (vs loading a bunch of 3rd party languages one at a time).

If you'd like to become the maintainer for this language then we're happy to create a new repository in the `highlightjs` organization and host the repository there.  That assumes you'd be open to reviewing issues and fixing bugs as they arise.  You're also welcome to host the repo yourself and we'll link to it from our README.

There isn't much in the way of how to docs on this yet, but there are sample repositories and it's honestly pretty easy.  We're still fleshing out an [official "spec" for 3rd party modules](https://github.com/highlightjs/highlight.js/issues/2328), but even after that's done it should require minimal changes from the examples we have already.  IE, if you wanted to get started now there isn't any big reason to wait.

My own `robots.txt` grammar repository is a great, simple example:

https://github.com/highlightjs/highlightjs-robots-txt

If you have any questions feel free to ask.

**Comment by yyyc514** [comment]
**Closing for inactivity;** (also finally cleaning all these old PRs out for the new year).

As mentioned above if you're interested in becoming the maintainer of a 3rd party module, please respond [whenever you find time] and we'd be happy to help you get started.

**Pull request closed without merging**